ros_ws/src/navigation_stack/msgs_converter/include/msgs_converter.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <ros/ros.h>
+
+#include <drive_msgs/drive_param.h>
+#include <geometry_msgs/Twist.h>
+
+class MSGSConverter
+{
+    public:
+    MSGSConverter();
+
+    private:
+    /**
+     * @brief ROS Handle
+     *
+     */
+    ros::NodeHandle m_node_handle;
+
+    /**
+     * @brief ROS Subscriber for servo position messages
+     * TOPIC: "cmd_vel"
+     */
+    ros::Subscriber m_command_velocity_subscriber;
+
+    /**
+     * @brief ROS Publischer for the velocity of left rear wheel
+     *
+     */
+    ros::Publisher m_to_car_control_publisher;
```

Comment:
Could this be renamed to something like `m_left_rear_wheel_velocity_publisher`? I didn't quite understand what this does. Maybe you can update the summary to explain why only the left rear wheel velocity needs to be published.